streamalert/shared/utils.py

```diff
@@ -143,3 +143,20 @@ def get_keys(data, search_key, max_matches=-1):
                 if val and isinstance(val, _CONTAINER_TYPES):
                     containers.append(val)
     return results
+
+def get_database_name(config):
+    """Get the name of the athena database using the current config settings
+    Args:
+        config (CLIConfig): Loaded StreamAlert config
+    Returns:
+        str: The name of the athena database
+    """
+    prefix = config['global']['account']['prefix']
```

Comment:
ha I was just thinking about doing something similar for this, and applied similar logic to other areas. good change